docs/docs/api/python.md

```diff
@@ -1,3 +1,3 @@
 # Python
 
-Write DBSP Programs using Python.
\ No newline at end of file
+Write Feldera Programs using Python.
```

Comment:
This can't be right either.
But this file is empty, so it doesn't really matter.